specification/consumption/resource-manager/Microsoft.Consumption/2017-11-30/consumption.json

```diff
@@ -155,7 +155,8 @@
             "description": "Can be daily or monthly",
             "in": "query",
             "required": true,
-            "type": "string"
+            "type": "string",
+            "enum": ["daily", "monthly"]
```

Comment:
please take a look at adding https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-enum and most likely modelAsString=true

Reply:
pushed the change